### ISSUE TITLE
[ALM Accelerator - BUG]: Download Solution Artifact task in export-to-git fails if solution size is >16MB

### ISSUE THREAD

**Issue by BJHF-power** [microsoft/coe-starter-kit#8251]
### Describe the issue

![image](https://github.com/microsoft/coe-starter-kit/assets/106766774/3fcf7b31-5829-4940-9c2b-c7d490ee51af)

### Expected Behavior

The solution should be downloadable regardless of file size. According to documentation, this should be accomplishable via chunked download: https://learn.microsoft.com/en-us/power-apps/developer/data-platform/file-column-data?tabs=sdk#download-the-file-in-chunks-using-web-api

### What component are you experiencing the issue with?

ALM Accelerator Pipelines

### What solution version are you using?

March

### Steps To Reproduce

1. Trigger an "export-solution_to_git-pipelines" run and provide an artifact URL for a solution artifact >16MB

### Anything else?

relevant code is in /PowerShell/dataverse-webapi-functions.ps1 starting at line 120

AB#3221

**Comment by v-bjaiswal** [comment]
Hi @BJHF-power ,thank you for raising the issue. We will look into this.

**Comment by OITIPower** [comment]
Hi @v-bjaiswal , 
I managed to get past this issue by changing two different things
1.	I added a backtick to the “$value” part of the artifact url (now looks like this "/`$value").  This is because powershell was interpreting that string as a variable, so we needed to escape it.  Without the $value parameter in the url, the pipeline will fail to download any Solution Artifact over 16MB
2.	In the next step “Unpack Solution Artifacts”, I replaced manual writing of bytes to a file, and replaced it with the “-Outfile” parameter for “Invoke-RestMethod.”  I did this because the response.value would be null if the solution was over 16MB
  
![PRForUnpackSolutionArtifacts](https://github.com/microsoft/coe-starter-kit/assets/169940326/193d91ba-5a6e-445d-9380-9cc7b4d30400)
